CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_ad_reporting v1.8.0
+Tiny release here!
+
+[PR #103](https://github.com/fivetran/dbt_ad_reporting/pull/103) just upticks the `microsoft_ads` dependency following the a major release. Refer to the Microsoft Ads v0.8.0 [release notes](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.8.0) for what exact updates have been included.
+
```

Comment:
Can we actually still have this be part of a sub section, just to keep the same structure as the other releases. For example something like the following:
```suggestion

[PR #103](https://github.com/fivetran/dbt_ad_reporting/pull/103) includes the following update:

## Dependency Updates
- Tiny update which upticks the `microsoft_ads` dependency following a recent major release. Refer to the Microsoft Ads v0.8.0 [release notes](https://github.com/fivetran/dbt_microsoft_ads/releases/tag/v0.8.0) for what exact updates have been included.
```